src/interfaces/index.ts

```diff
@@ -43,3 +43,6 @@ export * from './dissolution-confirmation-interface'
 export * from './todo-list-interface'
 export * from './steps-interface'
 export * from './table-header-interface'
+
+export interface StateFilingIF {
+}
```

Comment:
What is this for?

Reply:
Agreed, it's not needed.  I'll remove it.